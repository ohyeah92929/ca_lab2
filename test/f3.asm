.orig x3000
and r0, r0, #0 ; 0x5020, r0: 0
and r1, r1, r0 ; 0x5240, r1: 0
add r0, r0, #12; 0x102C, r0: 12
stw r0, r1, #10; 0x740a, mem[20]: 12
ldb r4, r0, #8; 0x2808, r4: 12
halt
target .fill x1111
.end