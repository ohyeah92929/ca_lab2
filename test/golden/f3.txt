
Memory content [0x3000..0x301c] :
-------------------------------------
 0x3000 (12288) : 0x5020
 0x3002 (12290) : 0x5240
 0x3004 (12292) : 0x102C
 0x3006 (12294) : 0x740a
 0x3008 (12296) : 0x2808
 0x300a (12298) : 0x1111
 0x300c (12300) : 0x0000
 0x300e (12302) : 0x0000
 0x3010 (12304) : 0x0000
 0x3012 (12306) : 0x0000
 0x3014 (12308) : 0x0000
 0x3016 (12310) : 0x0000
 0x3018 (12312) : 0x0000
 0x301a (12314) : 0x0000
 0x301c (12316) : 0x0000

Current register/bus values :
-------------------------------------
Instruction Count : 6
PC                : 0x0000
CCs: N = 0  Z = 2  P = 2
Registers:
0: 0x000c
1: 0x0000
2: 0x0000
3: 0x0000
4: 0x000c
5: 0x0000
6: 0x0000
7: 0x300a

